### DISCUSSION TITLE
Removing instant ways to distinguish dead and unconscious units.

### DISCUSSION THREAD

**Discussion by serraedoardo** [acemod/ACE3#8048] [General]
There has been interest lately in removing instant (and non-immersive) ways to detect if a unit is dead or just unconscious.
Here's a list of the various fixes proposed, but not yet (ACE version 3.13.5) implemented:

- ~~Inventory action on unconscious units: #8024 #8083~~
- Ability to carry/drag corpses: #7939 
- Prevent automatic unloading of dead or unconscious units: #7959 
- ~~Hide/Reimplement "Open Bag (Backpack)" action on unconscious units: #8084~~

I've started this discussion to summarize all of the aforementioned fixes and to expose other possible issues that might be unveiled, one of them being that collisions aren't enabled on corpses whilst they are on the head and torso of an unconscious unit.
I imagine that enabling collisions for all corpses might be very performance hitting, however, I feel that there should be an option to toggle on or off collision on corpses of dead players. If feasible of course.

**Comment by BaerMitUmlaut** [comment]
We don't turn off collisions, that's all in engine.

**Reply by serraedoardo** [reply]
I had the suspect it was an engine-side matter. Maybe it can be done with [enableCollisionWith](https://community.bistudio.com/wiki/enableCollisionWith) and [disableCollisionWith](https://community.bistudio.com/wiki/disableCollisionWith) commands.

**Reply by BaerMitUmlaut** [reply]
I doubt it.
